<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditExpense</class>
 <widget class="QDialog" name="EditExpense">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>20</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumDateTime">
    <datetime>
     <hour>23</hour>
     <minute>59</minute>
     <second>59</second>
     <year>9999</year>
     <month>8</month>
     <day>30</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>dd/MM/yyyy</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditComment">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>140</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelGoods">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>100</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Goods:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxCurrency">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>60</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditGoods">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelDate">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>20</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMoney">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>60</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Value:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelComment">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>140</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Comment:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMoneyValue">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>60</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonsOkCancel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonsOkCancel</sender>
   <signal>accepted()</signal>
   <receiver>EditExpense</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonsOkCancel</sender>
   <signal>rejected()</signal>
   <receiver>EditExpense</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
